ui/page/send/send_modal.go

```diff
@@ -0,0 +1,95 @@
+package send
+
+import (
+	"context"
+
+	"gioui.org/font"
+	"gioui.org/layout"
+
+	"github.com/crypto-power/cryptopower/ui/cryptomaterial"
+	"github.com/crypto-power/cryptopower/ui/load"
+	"github.com/crypto-power/cryptopower/ui/page/components"
+	"github.com/crypto-power/cryptopower/ui/values"
+)
+
+type PageModal struct {
+	*load.Load
+	*cryptomaterial.Modal
+
+	sendPage *Page
+
+	ctx       context.Context // page context
+	ctxCancel context.CancelFunc
+
+	okBtn cryptomaterial.Button
+
+	sourceWalletSelector *components.WalletAndAccountSelector
+}
+
+func NewPageModal(l *load.Load) *PageModal {
+	sm := &PageModal{
+		Load:  l,
+		Modal: l.Theme.ModalFloatTitle(values.String(values.StrSettings)),
+	}
+
+	sm.okBtn = l.Theme.Button(values.String(values.StrOK))
+	sm.okBtn.Font.Weight = font.Medium
+
+	// initialize wallet selector
+	sm.sourceWalletSelector = components.NewWalletAndAccountSelector(sm.Load).
+		Title(values.String(values.StrSelectWallet))
+	sm.setSelectedWallet()
+
+	sm.sendPage = NewSendPage(sm.Load)
```

Comment:
Wrapping a page in a modal doesn't look good to me. Can the original send page be used, maybe add a `showAsModal boolean` there to tweak the how it appears if that boolean is true?

Reply:
I will look into this.